### PR TITLE
Adding ligo-asimov

### PR DESCRIPTION
**Pull request by transientlunatic**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble be please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/ligo-asimov/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    - asimov
+    
+about:
+  home: https://github.com/transientlunatic/asimov
+  license: ISC
+  license_family: OTHER
+  license_file: LICENSE.txt
+  summary: 'Parameter Estimation automator for gravitational wave analyses.'
+  description: |
+    asimov is a python package used to construct automated bots to do various things with parameter esimation jobs for LIGO data analysis. There are a number of bits of code which are designed to make writing your own bot easier, and these are importable, but there are also pre-baked bots.
```
Comment (dopplershift) [inline recipes/ligo-asimov/meta.yaml:46]: "esimation" -> "estimation"

File: recipes/ligo-asimov/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (dopplershift) [inline recipes/ligo-asimov/meta.yaml:1]: Can you remove these comments at the top?

**Review by dopplershift — CHANGES_REQUESTED** [review]
Things look generally in good shape, just a couple minor things.

**Comment by dopplershift** [comment]
@duncanmmacleod are you willing to be a maintainer for this package recipe?

File: recipes/ligo-asimov/meta.yaml (hunk 1 above)
Comment (transientlunatic) [reply recipes/ligo-asimov/meta.yaml]: Thanks for spotting that!

File: recipes/ligo-asimov/meta.yaml (hunk 2 above)
Comment (transientlunatic) [reply recipes/ligo-asimov/meta.yaml]: Done.

**Comment by duncanmmacleod** [comment]
> @duncanmmacleod are you willing to be a maintainer for this package recipe?

Yes, please carry on.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**